packages/devtools/src/registry.ts

```diff
@@ -47,7 +47,7 @@ export function getAllForms() {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function registerField(field: FieldState<any>, type: string, vm: Maybe<ComponentInternalInstance>) {
+export function registerField(field: FieldState<any>, type: Ref<any> | string, vm: Maybe<ComponentInternalInstance>) {
```

Comment:
@logaretm Because fields are registered by the field controller, the type of field can change depending on when its registered. I believe this now must also be reactive.

Reply:
If it works as expected then it is fine, but I think when we set the `type` down there it evaluates the value at that time so it wouldn't be reactive if it does change.

Does this work as expected? if it does then disregard.

Reply:
Yea let's do that.